### PR TITLE
config swap pool

### PR DESCRIPTION
**Pull request by fiftyeightandeight**

### CODE REVIEW

File: clarity/Clarinet.toml [hunk 1]
```diff
@@ -976,3 +976,9 @@ depends_on = [ "trait-sip-010", "trait-ownable" ]
 [contracts.bridge-endpoint]
 path = "contracts/bridge/bridge-endpoint.clar"
 depends_on = ["trait-ownable", "trait-sip-010"]
+
+[contracts.amm-swap-pool]
+path = "contracts/pool/amm-swap-pool.clar"
+
+[contracts.token-amm-swap-pool]
+path = "contracts/pool-token/token-amm-swap-pool.clar"
```
Comment (zhigang1992) [inline clarity/Clarinet.toml:984]: @fiftyeightandeight these two contracts have to deps? is that expected?

Comment (zhigang1992) [reply clarity/Clarinet.toml:984]: https://github.com/alexgo-io/alex-v1/commit/d4798d367e8f0d0330a33811961d73d957da846e

Comment (fiftyeightandeight) [reply clarity/Clarinet.toml:984]: the latest version of clarinet removed this requirement (of `depends_on`). If you need them for our own use, I can certainly add them?

Comment (zhigang1992) [reply clarity/Clarinet.toml:984]: Yeah, we need them to determine the order of contract deployments. It'd be nice if we can keep them.

Comment (fiftyeightandeight) [reply clarity/Clarinet.toml:984]: Ok, I see you already made changes to the `dev`, so assume all good.

**Pull request merged**